flatten-array.yml

```diff
@@ -0,0 +1,4 @@
+---
+blurb: "Write a program that will take a nested array and returns a single with all values but null "
```

Comment:
Let's use _list_ instead of _array_. List feels like the more generic computer science-y term for this data structure.

Not all languages have nulls. Should we say _null or nil_ here?